addon/ng2/blueprints/ng2/files/src/tsconfig.json

```diff
@@ -8,9 +8,14 @@
     "moduleResolution": "node",
     "noEmitOnError": true,
     "noImplicitAny": false,
+    "outDir": "../dist/",
     "rootDir": ".",
     "sourceMap": true,
     "sourceRoot": "/",
     "target": "es5"
-  }
+  },
+  "filesGlob": [
```

Comment:
this is not nice. Only atom understands this property, not tsc.

Your options are
* use exclude, meaning all `.ts` and `.d.ts` files in this directory and children
* use files =[] which means you need to specify the application entry point(s) as well as the typings/browser.d.ts file (or the typings/main.d.ts file if targetting node runtime)